core/network/types/status.hpp

```diff
@@ -42,7 +42,7 @@ namespace kagome::network {
     /**
      * Best block number.
      */
-    BlockNumber best_number;
+    uint32_t best_number;
```

Comment:
Why?

Reply:
Substrate expects 4 byte unsigned in this field of ```Status``` message.